spec/features/admin/edit_organisation_details_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'features/support/sign_up_helpers'
+
+describe 'editing an organisations details' do
+  let!(:admin_user1) { create(:user, organisation: organisation) }
+  let!(:admin_user2) { create(:user) }
+  let!(:organisation) { create(:organisation, name: "TestMe & Company", service_email: "testme@gov.uk") }
+
+  context 'when visiting the organisations settings page before any changes' do
+    before do
+      sign_in_user admin_user1
+      visit organisation_path(organisation)
+    end
+
+    it 'has the current company name and service email' do
+      expect(page).to have_content("TestMe & Company")
+      expect(page).to have_content("testme@gov.uk")
+    end
+  end
+
+  context 'when visiting the edit organisations settings page' do
+    before do
+      sign_in_user admin_user1
+      visit edit_organisation_path(organisation)
+    end
+
+    it 'allows the user to change their company name and service email' do
+      fill_in 'Organisation name', with: "New Company Name"
+      fill_in 'Service email', with: "NewServiceEmail@gov.uk"
+      click_on 'Save'
+      expect(page).to have_content("New Company Name")
```

Comment:
This test might be too permissive, but I'm not sure (depends on how `have_content` behaves)

Consider the scenario where someone accidentally breaks the validation on this model, and the save always fails. When I hit 'Save' on this page, the save will fail, and I'll be on the `new` view still, but with what I typed in still present. 

If `have_content` checks those input elements, this'll pass, even though the functionality is broken.

If `have_content` doesn't look in input elements, then this isn't a problem, and this test seems fine.

Reply:
Bit confused by this. Speak to you face to face about it maybe